docs/shape-providers.md

```diff
@@ -164,6 +164,30 @@ public record MyPoco<T>(T Value)
 }
 ```
 
+#### Associated types
+
+The `TypeShape` attribute can also be used to specify associated types for the target type.
+This provides a Reflection-free way to activate an associated type via its default constructor
+given its @PolyType.Abstractions.ITypeShape.
+For example, serializers may need to jump from a type shape to its converter.
+
+[!code-csharp[](../samples/CSharpSamples/AssociatedTypes.cs#TypeShapeOneType)]
+
+@PolyType.Abstractions.ITypeShape.GetAssociatedTypeFactory*?displayProperty=nameWithType is the method to use to obtain a factory for instances of the associated type.
```

Comment:
Is this syntax for linking to API docs generated by docfx? I didn't that was possible, pretty cool.

Reply:
Yes, that is one of [several allowed forms](https://dotnet.github.io/docfx/docs/links-and-cross-references.html#markdown-links).